src/fileioc/fileioc.h

```diff
@@ -382,6 +382,38 @@ uint8_t ti_RclVar(const uint8_t var_type, const char *var_name, void **data_stru
  */
 bool ti_ArchiveHasRoom(uint24_t num_bytes);
 
+
+/**
+ * Set routine to run after a garbage collect.
+ * @param routine Routine to run following a garbage collect. NULL sets it to do nothing.
+ * @note If your program uses graphx, pass gfx_Begin to setup graphics after the garbage collect finishes.
+ * */
+void ti_SetPostGCHandler(void (*routine)(void));
+
+/**
+ * Set routine to run before a garbage collect.
+ * @param routine Routine to run preceeding a garbage collect. NULL sets it to do nothing. If this routine returns true, the variable will not be archived.
+ * @note Useful for cleanup. If your program uses graphx, run gfx_End inside the passed function to set OS graphics mode to avoid corrupted graphics during the garbage collect.
+ * @code
+ *    bool pre_gc_handler(void){
+ *          gfx_End();
+ *          return 0;
```

Comment:
I was about to leave a comment that this documentation would need to be updated like this, but it looks like you were already on top of it. The one change I'd request, though, is that `0` be changed to `false`.

Reply:
Edited